src/include/fcoe-client/complex.rb

```diff
@@ -447,11 +435,7 @@ def HandleInterfacesDialog(id, event)
             # if /etc/sysconfig/network/ifcfg-<if>.<vlan> already exists
             # call 'ifup' for the interface (creates /proc/net/vlan/<if>.<vlan>)
             if FileUtils.Exists(ifcfg_file)
-              cmd_ifup = Builtins.sformat(
-                "ifup %1.%2",
-                dev_name,
-                Ops.get_string(card, "vlan_interface", "")
-              )
+              cmd_ifup = Builtins.sformat("ifup %1.%2", dev_name, vlan_interface)
```

Comment:
we have ```SetIfaceUp``` in yast2-network ;-)

Reply:
Having learned about it, well, it is such a lone function (nobody uses it!) that I don't feel comfortable using it.
BTW it does not document its return value: http://www.rubydoc.info/github/yast/yast-network/Yast%2FNetworkRoutinesInclude%3ASetIfaceUp

Reply:
well it used to be used. Currently it is replaced by something more friendly to wicked.